releases.md

```diff
@@ -41,12 +41,12 @@ Further documentation available:
 
 ## Release
 
-### v0.46
+### v0.47 (LTS)
 
-- **Latest Release**: [v0.46.0][v0-46-0] (2023-03-17) ([docs][v0-46-0-docs], [examples][v0-46-0-examples])
-- **Initial Release**: [v0.46.0][v0-46-0] (2023-03-17)
-- **Estimated End of Life**: 2023-04-17
-- **Patch Releases**: [v0.46.0][v0-46-0]
+- **Latest Release**: [v0.47.0][v0-47-0] (2023-04-25) ([docs][v0-47-0-docs], [examples][v0-47-0-examples])
+- **Initial Release**: [v0.47.0][v0-47-0] (2023-04-25)
+- **End of Life**: 2024-04-24
```

Comment:
Thanks Andrea! I think I misunderstood our support policy for LTS releases-- it sounds like it's exactly 1 year because there's overlap in LTS releases? Should we update https://github.com/tektoncd/community/blob/main/releases.md#support-policy to read "The Tekton project maintains four release branches for each project, created one every three months, which results in a overall support window of one year for each of these releases." (i.e. remove "approximately" one year?)

Reply:
Oh, I thought it was set to 1 year, but I shouldn't have trusted memory :D
We can also leave "approximately", and EOL an LTS once the next release goes out so that we have the same number of releases around.